### DISCUSSION TITLE
请问如何将本机的Socks代理用这个工具打洞出去呢？

### DISCUSSION THREAD

**Discussion by LordHumphrey** [MikeWang000000/Natter#60] [Q&A]
现在搭了一个socks代理，想打洞出去，但是似乎填写本机地址:port,这种形式打洞不行啊。折腾一回才过来问的，谢谢老哥~

**Comment by MikeWang000000** [comment]
首先，您使用的是哪个版本？在哪里运行的Natter？(路由器/路由器下的服务器?) 当前网络NAT显示如何？
如果不太能确定这些信息，可以进群和热心网友一起探讨讨论：）

**Comment by LordHumphrey** [comment]
> 首先，您使用的是哪个版本？在哪里运行的Natter？(路由器/路由器下的服务器?) 当前网络NAT显示如何？ 如果不太能确定这些信息，可以进群和热心网友一起探讨讨论：）

并不是路由，是运行在虚拟机上，代理另一台虚拟机上的服务，当前网络能够通过nat check，0.9版本的，谢谢

**Comment by MikeWang000000** [comment]
使用以下命令，能否在公网上访问测试用http服务？
```
python natter.py -t 3456
```
